src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -111,7 +111,7 @@ private String getParameter(HttpServletRequest request, String name, String defa
   private int getId(HttpServletRequest request) {
     int fail = -1;
     try {
-      return Integer.parseInt(request.getParameter("id"));
+      return Integer.parseInt(request.getParameter("school-id"));
```

Comment:
Why did you change this? I am pretty sure it should be just id

Reply:
fixed

Reply:
I think it may have been caused by a rebase issue from earlier. Should be fixed now